tracing/src/main/java/com/palantir/tracing/Tracers.java

```diff
@@ -104,6 +106,11 @@ public static Runnable wrap(Runnable delegate) {
         return new TracingAwareRunnable(delegate);
     }
 
+    /** Like {@link #wrap(Callable)}, but for Guava's FutureCallback. */
+    public static <V> FutureCallback<V> wrap(FutureCallback<V> delegate) {
+        return new TracingAwareFutureCallback<>(delegate);
```

Comment:
Should this take an operation name? Once #71 merges DeferredTracer can take an operation name. I'd prefer we we required an operation name for new methods, and did not provide a method using a default.

Reply:
Sure. Can wait until #71 merges and then rebase to use the DeferredTracer with the operation name.